tests/Functional/Net/SFTPWrongServerTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * @author Vladimir Gaydamaka <vladimir.gaydamaka@gmail.com>
+ * @copyright 20019 Vladimir Gaydamaka
+ * @license   http://www.opensource.org/licenses/mit-license.html  MIT License
+ */
+
+use phpseclib\Net\SFTP;
+use PHPUnit\Framework\Error\Warning;
+
+class SFTPWrongServerTest extends Functional_Net_SFTPTestCase
+{
+    public function setUp()
+    {
+      parent::setUp();
+      $this->scratchDir = uniqid('phpseclib-sftp-scratch-');
+
+      $this->sftp = new SFTP($this->getEnv('SSH_WRONG_HOSTNAME'));
+    }
+
+    public function testLogin()
+    {
+      try {
+        $this->sftp->login(
```

Comment:
Replace for `(new SFTP('dummy-hostname'))->login('login', 'password')`. Do not use env variables since they have no sense here.